atlas/tasks.py

```diff
@@ -138,11 +138,56 @@ def code_deploy(item):
     except GitCommandError:
         log.error('Code | Checkout | Cannot checkout requested tag, check value.')
     if item['meta']['is_current']:
+        # Create 'current' directory for code item
         code_operations.update_symlink_current(item)
-        log.debug('Code deploy | Symlink | Is current')
-    
-    if item['meta']['code_type'] == 'static':
-        code_operations.deploy_static(item)
+        # Case for profiles
+        if item['meta']['code_type'] == "profile":
```

Comment:
I think the three of these can be combined into a single statement since `code_operations.update_symlink_profile` handles all types.

Something like
```
            log.debug('Code deploy | Packages Symlink | Is current')
            # Query for packages
            package_query = 'where={"meta.code_type":{"$in":["module","library","theme"]},"meta.is_current":true}'
            package_items = utilities.get_eve('code', package_query)

            if package_items:
                for package in package_items['_items']:
                    log.info('Code deploy | Adding package symlinks to profile')
                    # modules is a list of current packages, item is the profile object.
                    code_operations.update_symlink_profile(package, [item])
```